CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Contributing to MusicBrainz Server
+
+Thank you for considering contributing to MusicBrainz Server.
+
+Get started with https://musicbrainz.org/doc/Development first,
+then continue reading with below additional guidelines.
+
+## Submitting changes
+
+### Commit message
+
+Please follow the generally accepted [seven rules of a great Git
+commit message](https://chris.beams.io/posts/git-commit/#seven-rules):
+
+1. Separate subject from body with a blank line
+1. Limit the subject line to 50 characters
+1. Capitalize the subject line
+1. Do not end the subject line with a period
+1. Use the imperative mood in the subject line
+1. Wrap the body at 72 characters
+1. Use the body to explain _what_ and _why_ vs. _how_
+
+Additionally, start the subject line with ticket reference if applicable.
+
+### Pull request
+
+#### Ticket
+
+If your change is large or relevant to users, it should have a ticket, see
+[tickets](https://tickets.metabrainz.org/issues/?jql=project%20%3D%20MBS),
```

Comment:
https://tickets.metabrainz.org/browse/MBS might be slightly better than a search here. I resolved some minor comma splices, too

```suggestion
should have a ticket in [our issue tracker](https://tickets.metabrainz.org/browse/MBS).
Create one if necessary.
```

Reply:
Applied in https://github.com/metabrainz/musicbrainz-server/pull/1345/commits/3fe58fc29b62ae96b1115f042aa8a026033bbd54#diff-6a3371457528722a734f3c51d9238c13R33